src/index.ts

```diff
@@ -56,6 +61,13 @@ if (config.health.healthz.enabled) {
     }
 
     config.RUNTIME.client = client;
+    // probably better to change to a config value
```

Comment:
This will all need moving to setup code in a testing module

Reply:
Yes, please :)